src/vm/dllimport.cpp

```diff
@@ -6563,6 +6563,21 @@ class LoadLibErrorTracker
     DWORD   m_priorityOfLastError;
 };  // class LoadLibErrorTracker
 
+static HMODULE LocalLoadLibraryHelperInternal(LPCWSTR name, DWORD flags)
```

Comment:
The call to `LoadLibraryEx` was ifdefed for Unix even before your change using `FEATURE_PAL`. If you just keep the original `FEATURE_PAL` ifdef in place, without introducing yet another wrapper layer, it should work just fine.